Enhancements for supporting LIS hotfix release

iamshital opened the pull request.

Commit:
RH7: Enhancement of install & upgrade script for lis-hotfix-pipeline support

Commit:
lis-hotfix-build support: RH7: Enhancement of install & upgrade script

Commit:
lis-hotfix-build support: RH7: Enhancement of install & upgrade script

iamshital committed:
lis-hotfix-build support: Dynamic generation of ips.sh & pbuild file

iamshital committed:
Added PS script to dynamically create ips.sh and pbuild files

iamshital committed:
lis-hotfix-build support: Dynamic generation of ips.sh & pbuild file

iamshital committed:
Updated copyall.sh

iamshital committed:
lis-hotfix-build support: Updated copyall.sh to clean and copy RPMs

iamshital committed:
lis-hotfix-build support: Updated copyall.sh to clean and copy RPMs

iamshital committed:
lis-hotfix-build support: Support to generate errata kernel version list

iamshital committed:
lis-hotfix-build support: Support to generate errata kernel version list

iamshital committed:
Added support to create kernel version list

Commit:
lis-hotfix-build support: Enhancement of copyall.sh to cleanup and copy SRPMs

Commit:
lis-hotfix-build support: Enhancement of copyall.sh to cleanup and copy SRPMs

iamshital committed:
Merge remote-tracking branch 'john/lis-hotfix-demo' into lis-hotfix-demo

johnsongeorge-w commented:
@DivyaJyothiKasam : Review the changes.

santoshx commented on LISISO/RPMS70/install.sh:
```diff
@@ -5,14 +5,18 @@
+	if ! IsInstalledKernelOlderThanErrataKernel $ekernel;then
+		break
+	fi
+	update_number=$(expr $update_number + 1)
```
expr is deprecated. You may consider rewriting this using $((..))

santoshx commented on LISISO/RPMS70/install.sh:
```diff
@@ -5,14 +5,18 @@
-else
-	installbuildrpm 1
-fi
+#List of errata kernel that needs special handling due to KABI changes
```
May be a space after #

santoshx commented on LISISO/RPMS73/install.sh:
```diff
@@ -1,21 +1,22 @@
+for ekernel in $ekernel_list;do
+	if ! IsInstalledKernelOlderThanErrataKernel $ekernel;then
+		break
+	fi
```
Mix of tabs and spaces

santoshx reviewed (approved):
I see some minor spacing errors other than looks good.

Commit:
RH7: Enhancement of install & upgrade script for lis-hotfix-pipeline support

This commit optimizes the commit 1473f0284ab6
Enhances: 1473f0284a (RH7: Enhancement of install & upgrade script for lis-hotfix-pipeline support)

johnsongeorge-w commented:
Addressed all the comments. I have enhanced the install and upgrade script to use the commonfunction script for determining the update number (https://github.com/LIS/lis-rpm-build-pipeline/commit/cc24c6d37eda8e65448ec4957fe86dc66fcd5525). 
The same changes is pulled in and PR is updated.

johnsongeorge-w commented:
New PR #61 raised.
